parlai/core/torch_generator_agent.py

```diff
@@ -377,7 +377,10 @@ def __init__(self, opt, shared=None):
                 print('[ Saving dot beam logs in {} ]'.format(self.beam_dot_dir))
 
             self.build_criterion()
-            self.build_model()
+            with torch.random.fork_rng(range(torch.cuda.device_count())):
+                torch.manual_seed(42)
```

Comment:
should we make this seed configurable? or is that too dangerous...

Reply:
probably

Reply:
left a TODO comment